frontend/src/routes/CriticalQuestions/index.js

```diff
@@ -47,13 +49,17 @@ function CriticalQuestions(props) {
     const [locationCount, setLocationCount] = useState(0)
 
     const [selectedProblems, setMedicalProblems] = useState([]);
+    const dispatch = useDispatch();
 
     const handleMedicalProblemChange = (event) => {
```

Comment:
See, it's this kind of helpers that we could be simplifying into a more compact method. Check out the `handleFormChange` on the `<SignIn/>` & `<SignUp/>` views or let me know if you want a more thorough explanation!